pkg/cloud/openstack/assets/rbac/serviceaccount.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: openshift-cloud-controller-manager
+  name: openstack-cloud-controller-manager
```

```suggestion
  name: cloud-controller-manager
```

Let's centralize the naming and move it to manifests for other providers to use.